homeassistant/components/tts/__init__.py

```diff
@@ -590,10 +671,11 @@ async def _async_get_tts_audio(
 
         This method is a coroutine.
         """
-        if options is not None and ATTR_AUDIO_OUTPUT in options:
-            expected_extension = options[ATTR_AUDIO_OUTPUT]
-        else:
-            expected_extension = None
+        options = options or {}
+        blocking = options.get(ATTR_BLOCKING, "true") != "false"
```

Comment:
Why are these strings? Does that mean that the conversion from options to a dict failed? It's not always coming from a media source URI so that could break now ?

Reply:
Since we now control the extension, we never have to block anymore. The blocking will happen when you fetch the content (which is when ESPHome will wait). We should remove `ATTR_BLOCKING` completely.

Reply:
Removed `ATTR_BLOCKING`. It seemed that "yarl" did not support booleans in the URL serialization for some reason, so I went with strings originally.